apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

```diff
@@ -40,14 +40,50 @@ public void setup() {
             .map(r -> (Beneficiary) r)
             .findFirst()
             .get();
-    beneficiary.setLastUpdated(new Date());
+    // beneficiary.setLastUpdated(new Date());
```

Comment:
I'd remove this if it isn't necessary.  This _should_ play into the end result though I would expect?